templates/feature/project.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
    * Year that the project was created.
    */
   createYear: '<<year>>',
+  /**
+   * Date that the project was last rebuilt. The updatedDate is the created date before the first build.
+   */
```

Comment:
Small thing, but since this is automatically changed, we could note that in the comment so someone wouldn't think they could edit it.